README.md

```diff
@@ -1,24 +1,19 @@
-Chitter Challenge
+Weekend 4: Chitter Challenge
 =================
+Features:
+-----------------
 
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
-
-Challenge:
--------
+Aim: To achieve the first 4 user stories
```

Comment:
Clear approach, I like you've thought about and set yourself a clear target to achieve.